src/modules/user/dto/rate.request.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { IsNumber, IsString } from 'class-validator';
+
+export class RateRequestDto {
```

Comment:
I dont understand when client will ask you for rate based on id and rate? I assume its mean that client already have needed data

Reply:
this dto used to make rate to user, not to ask the rate. Mb "id" prop confuse You, changed to "rate_target_id".
Request in this name means that we wait such data in request when user wants to rate another, not that we request some data from db

Reply:
when client will know rate_target_id? rates table saved in the backend, just to calculate the average rate properly